common/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+#![allow(
```

Comment:
why allow unknown_clippy_lints here?

Reply:
Looks like this link is added in another PR